### DISCUSSION TITLE
30 day free trial

### DISCUSSION THREAD

**Discussion by JeroenHaverkort** [plausible/analytics#3853] [Self-Hosted Support] [answered]
i recently setup plausible v2.0 self hosted on my local machine for testing. i received this mail stating that there is a 30 day trial. My question is what happens after the 30 days?
![image](https://github.com/plausible/analytics/assets/93723087/2927d5c2-5936-4ab6-afa0-3d0af6473361)

**Comment by ruslandoga** [comment, marked as answer]
👋 @JeroenHaverkort 

This looks like a bug. I'll look into it and try to reproduce with v2.1.0-rc.0.

**Reply by madanyang** [reply]
I don't think it is a bug, but rather a limitation how the templates are used. If you can change the email templates, and the logic when and how emails are sent then this trial email would never present itself in a self hosted environment as it doesn't make sense. See #2684 where customization was asked.

Currently the only way possible is to build your own docker with changed templates. So the use of docker image provided by the project becomes irrelevant. A better approach would be to enable editing the templates, mail flow logic.

**Reply by ruslandoga** [reply]
👋 @madanyang 

We are planning to remove emails like in the OP in v2.1.0 and generally make it clear that emails come from Plausible CE builds and not the cloud. So it is kind of unexpected if a trial email is still being sent out.

Currently there are no plans to allow template customizations.

**Comment by jacopobonomi** [comment]
Same problem in 
https://github.com/plausible/analytics/blob/master/lib/plausible_web/templates/site/new.html.heex#L17
<img width="531" alt="Screenshot 2024-09-04 alle 12 32 11" src="https://github.com/user-attachments/assets/06f20d49-4b35-463f-a5bb-46d47a025dfa">
Probably is better remove all trial ref in OP project, moving all logic in a separate library.

**Reply by ruslandoga** [reply]
👋 @jacopobonomi 

Thank you the report. I don't think we can move it to a separate library, but we can probably standardize on a single function that checks if the user is on trial and disable that check in CE.

**Reply by jacopobonomi** [reply]
@ruslandoga Sounds good!
